hw/ip/hmac/data/hmac.hjson

```diff
@@ -290,9 +290,10 @@
               ''',
         count: "NumWords",
         cname: "HMAC",
-        swaccess: "ro",
-        hwaccess: "hwo",
+        swaccess: "rw",
+        hwaccess: "hrw",
```

Comment:
Would it make sense to mention the in the register description that writes by software are ignored unless the SHA core is disabled? Actually, don't you need to update the tags for the automated CSR tests as well?

Reply:
Thanks for the suggestion, I extended the description for the digest registers and the message length registers.

I don't think these registers need exclusion tags for the automated CSR tests because they will be readable and writable as expected during the automated CSR tests: `CFG.sha_en` is excluded from writes by the automated CSR tests, so it will remain `0` during these tests.

Reply:
This makes a lot of sense, thanks for explaining.